### PR TITLE
Update docs/src/jump/README.md

### PR DESCRIPTION
**Pull request by odow**
Here are some changes, but others might be needed. I think this is going to take a few rounds of iteration. I don't think the footnotes will work.

x-ref https://github.com/jump-dev/JuMP.jl/pull/3561

### CODE REVIEW

**Comment by DimitriAlston** [comment]
I can remove the reference to the footnotes, if that's the issue.

**Comment by odow** [comment]
The main issue was fixing the `/>` closing tags for `img` and `p`. You could also just use `[1]` etc without links to the bottom?

**Comment by DimitriAlston** [comment]
I sent you a message on Slack. I think it would be easier to talk about this there, unless you have another preference.

File: docs/src/jump/README.md [hunk 1]
```diff
@@ -1,46 +1,69 @@
-<img src="https://github.com/PSORLab/EAGO.jl/blob/master/docs/src/assets/logo.png" width="75%" height="75%">
+<img src="https://github.com/PSORLab/EAGO.jl/blob/master/docs/src/assets/logo.png" width="75%" height="75%"/>
```
Comment (odow) [inline docs/src/jump/README.md:1]:
````suggestion
```@raw html
<img src="https://github.com/PSORLab/EAGO.jl/blob/master/docs/src/assets/logo.png" width="75%" height="75%"/>
```
````

**Pull request merged**